js_modules/dagit/packages/core/src/assets/AutoMaterializePolicyPage/AssetAutomaterializePolicyPage.tsx

```diff
@@ -216,45 +232,31 @@ export function getEvaluationsWithSkips({
 export const PAGE_SIZE = 30;
 function LeftPanel({
   evaluations,
+  evaluationsIncludingEmpty,
   paginationProps,
   onSelectEvaluation,
   selectedEvaluation,
-  currentEvaluationId,
-  isFirstPage,
-  isLastPage,
-  isLoading,
 }: {
   evaluations: EvaluationType[];
+  evaluationsIncludingEmpty: any[];
   paginationProps: ReturnType<typeof useEvaluationsQueryResult>['paginationProps'];
   onSelectEvaluation: (evaluation: EvaluationType) => void;
-  selectedEvaluation?: EvaluationType;
-  currentEvaluationId: number | null;
-  isFirstPage: boolean;
-  isLastPage: boolean;
-  isLoading: boolean;
+  selectedEvaluation?: any;
```

Comment:
I'm not familiar with ts types, but this could be a union of the EvaluationType and the `__typename: 'no_conditions_met' as const` type